### PR TITLE
Change readme to reflex

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
-Major Changes:
-+ Our domain will switch from https://pynecone.io/ -> https://reflex.dev/
-+ Our pip install will switch from pip install pynecone -> pip install reflex
-+ Our Github repo will switch from https://github.com/pynecone-io/pynecone -> https://github.com/reflex-dev/reflex
-+ Our library will change from import pynecone as pc to import reflex as rx
-
-We will have a feature freeze until we migrate our code and website to the new name.
-
-Our goal is to complete the transition by June 30. Thanks for your support - please message on Discord if you have any questions or concerns!
++ Searching for Pynecone? You are in the right repo. Pynecone has been renamed Reflex. +
```
Comment (picklelo) [inline README.md]: renamed to Reflex

File: README.md [hunk 2]
```diff
@@ -1,31 +1,20 @@
...
 **✨ Performant, customizable web apps in pure Python. Deploy in seconds.**
 
-📑 [Docs](https://pynecone.io/docs/getting-started/introduction) &nbsp; 📱 [Component Library](https://pynecone.io/docs/library) &nbsp; 🖼️ [Gallery](https://pynecone.io/docs/gallery) &nbsp; 🛸 [Deployment](https://pynecone.io/docs/hosting/deploy)
+📑 [Docs](https://reflex.dev/docs/getting-started/introduction) &nbsp; 📱 [Component Library](https://reflex.dev/docs/library) &nbsp; 🖼️ [Gallery](https://reflex.dev/docs/gallery) &nbsp; 🛸 [Deployment](https://reflex.dev/docs/hosting/deploy)
 
-[![PyPI version](https://badge.fury.io/py/pynecone.svg)](https://badge.fury.io/py/pynecone)
+[![PyPI version](https://badge.fury.io/py/reflex.svg)](https://badge.fury.io/py/reflex)
 ![tests](https://github.com/pynecone-io/pynecone/actions/workflows/build.yml/badge.svg)
-![versions](https://img.shields.io/pypi/pyversions/pynecone-io.svg)
+![versions](https://img.shields.io/pypi/pyversions/reflex-io.svg)
```
Comment (picklelo) [inline README.md]: This may need to be `reflex`

File: README.md [hunk 3]
```diff
@@ -40,42 +29,42 @@ Our goal is to complete the transition by June 30. Thanks for your support - ple
 
 ## 📦 1. Install
 
-Pynecone requires the following to get started:
+reflex requires the following to get started:
```
Comment (picklelo) [inline README.md:33]: Capitalize Reflex

**Review by picklelo — COMMENTED** [review]
Looks good - some places we are missing capitalization. You can replace Pynecone with Reflex first and then pynecone with reflex

**Review by masenf — APPROVED** [review]

**Pull request merged**